includes/namespace.php

```diff
@@ -275,6 +271,22 @@ function deactivate( $network_wide ) {
 register_activation_hook( WEBSTORIES_PLUGIN_FILE, __NAMESPACE__ . '\activate' );
 register_deactivation_hook( WEBSTORIES_PLUGIN_FILE, __NAMESPACE__ . '\deactivate' );
 
+
+if ( ! $web_stories_compatibility->check_required_files() || ! $web_stories_compatibility->check_php_version() || ! $web_stories_compatibility->check_wp_version() ) {
```

Comment:
Instead of doing all these things (e.g. long array syntax) here in this file, why not run the PHP compatibility check in `web-stories.php`?

IMHO in `namespace.php` we should not be writing PHP 5.3 code

Reply:
Doing it this way, makes it easier to unit test. I think it is a small price to pay for php 5.3 support.

Reply:
Then I'd much rather drop the PHP version checks entirely tbh

Reply:
I really think it is worth it. These are small tweaks that could have massive impacts on users that white screen of death there WordPress installs because of this plugin.

Reply:
@swissspidy I can also just revert the short array changes. Then we could support 5.4+. 
It is just a matter of reverting https://github.com/google/web-stories-wp/pull/5071/commits/989d16b821f4b5bd29c7347b517a9caa656a8ae7 https://github.com/google/web-stories-wp/pull/5071/commits/de8a3e1100ed4db99369d1ebbfd0614daf595510